docs/reference/islandora_defaults_reference.md

```diff
@@ -0,0 +1,395 @@
+
+
+## Migrations
+
+
+| Name | 
+|---|
+|Tags migration for islandora_defaults feature (islandora_defaults_tags)| 
+
+Contents of the islandora_defaults_tags migration: 
+
+|Vocabulary|Term name|Description|External Uri|
+|---|---|---|---|
+|islandora_display|"Open Seadragon"|"Display using the Open Seadragon viewer"|http://openseadragon.github.io|
+|islandora_display|"PDFjs"|"Display using the PDF.js viewer"|http://mozilla.github.io/pdf.js|
+|resource_types|"Collection"|"An aggregation of resources"|http://purl.org/dc/dcmitype/Collection|
+|resource_types|"Dataset"|"Data encoded in a defined structure"|http://purl.org/dc/dcmitype/Dataset|
+|resource_types|"Image"|"A visual representation other than text"|http://purl.org/dc/dcmitype/Image|
+|resource_types|"Interactive Resource"|"A resource requiring interaction from the user to be understood, executed, or experienced"|http://purl.org/dc/dcmitype/InteractiveResource|
+|resource_types|"Moving Image"|"A series of visual representations imparting an impression of motion when shown in succession"|http://purl.org/dc/dcmitype/MovingImage|
+|resource_types|"Physical Object"|"An inanimate, three-dimensional object or substance"|http://purl.org/dc/dcmitype/PhysicalObject|
+|resource_types|"Service"|"A system that provides one or more functions"|http://purl.org/dc/dcmitype/Service|
+|resource_types|"Sound"|"A resource primarily intended to be heard"|http://purl.org/dc/dcmitype/Sound|
+|resource_types|"Still Image"|"A static visual representation"|http://purl.org/dc/dcmitype/StillImage|
+|resource_types|"Software"|"A computer program in source or compiled form"|http://purl.org/dc/dcmitype/Software|
+|resource_types|"Text"|"A resource consisting primarily of words for reading"|http://purl.org/dc/dcmitype/Text|
+
+
+core.base_field_override.node.islandora_object.changed.yml
+core.base_field_override.node.islandora_object.created.yml
+core.base_field_override.node.islandora_object.menu_link.yml
+core.base_field_override.node.islandora_object.path.yml
+core.base_field_override.node.islandora_object.promote.yml
+core.base_field_override.node.islandora_object.status.yml
+core.base_field_override.node.islandora_object.sticky.yml
+
+
+core.base_field_override.node.islandora_object.title.yml
+core.base_field_override.node.islandora_object.uid.yml
+
+
+
+## User Roles
+
+The role fedoraadmin (machine name: `fedoraadmin`) is defined.
+
+## Hooks, code, etc.
+- Add `'relators' => 'http://id.loc.gov/vocabulary/relators/`' to the RDF namespaces.
+
+
+## Fields
+
+This is at the end of this document because it's not particularly useful yet - there's so much information it's not clear what needs to be here.
```

Comment:
Maybe you don't need to list out every field setting.  The new fields are definitely important.  The field storage basically covers all the new fields.